Scala/FizzBuzz_sathiyajith.scala

```diff
@@ -0,0 +1,19 @@
+
+object FizzBuzz{  
+    def fizzBuzz() = {
+      for( x <- 1 to 100)
+    {
+  if (x % 15 == 0)
+    println("FizzBuzz")
+  else if (x % 3 == 0)
+    println("Fizz")
+  else if (x % 5 == 0)
+    println("Buzz")
+  else
+    x
+    }
+}
+    def main(args:Array[String]){  
+       fizzBuzz()
+    }  
+}  
```

Comment:
the exact same solution already submitted for Scala, You can modify your approach and raise the PR. https://github.com/ashish-r/Fizz-Buzz/blob/main/Scala/FizzBuzzWithScala.scala

Reply:
I have added new code for FizzBuzz. Thanks